llpc/test/shaderdb/partial_pipeline_cache_test/partial_pipeline_cache_test.ll

```diff
@@ -0,0 +1,11 @@
+; Test that partial pipeline cache is handled correctly.
```

Comment:
Can you briefly explain what the correct (expected) behavior is? This test is a bit unusual because it uses multiple input files, while the rest of the shaderdb tests rely on a single input which defines the expected behavior as well.

Reply:
Added. Please check if it is suitable.